AndroidManifest.xml.template

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    package="com.unity3d.player"
+    android:installLocation="preferExternal">
+
+    <supports-screens
+        android:anyDensity="true"
+        android:largeScreens="true"
+        android:normalScreens="true"
+        android:smallScreens="true"
+        android:xlargeScreens="true"/>
+
+    <!-- Some mediated networks do not include a manifest with permissions declaration within the aar they distribute. -->
+
+    <!-- The permissions below are required by some of mediated networks. Though they might still work without them, it is not guaranteed -->
+
+    <!-- Required by all mediation networks -->
+    <uses-permission android:name="android.permission.INTERNET"/>
+    <!-- Required by AdColony, AppLovin, (and optionally by FairBid 2) -->
```

Comment:
I believe not network requires this permission currently. It's just an optional permission.

Reply:
I refactored it in this commit
7735040